### PR TITLE
Current page & disable block area preview

### PR DESCRIPTION
**Pull request by jonom**
2 small improvements

### CODE REVIEW

File: code/dataobjects/Block.php [hunk 1]
```diff
@@ -428,6 +428,14 @@ public function CSSClasses($stopAtClass = 'DataObject')
     }
 
     /**
+     * Access current page scope from Block templates with $CurrentPage
+     * @return SiteTree | Controller
+     */
+    public function getCurrentPage() {
+        return Director::get_current_page();
```
Comment (anselmdk) [inline code/dataobjects/Block.php]: I've solved this with `Controller::curr();`, not sure what's the difference...

Comment (jonom) [inline code/dataobjects/Block.php]: `Director::get_current_page()` only gives you the base SiteTree/Page object, not the controller so it's not as useful actually. You wouldn't be able to use any page controller methods with what I've done here.

Probably is a good idea to use `Controller::curr()` instead... `getCurrentPage` doesn't quite seem semantic as a name choice in that case but `$CurrentPage` seems more intuitive to me to use in templates than `$CurrentController` so maybe it's okay?

Comment (anselmdk) [inline code/dataobjects/Block.php]: I used `getPageController`, not sure what's best. I see you already discussed this in https://github.com/sheadawson/silverstripe-blocks/issues/69, so I'm happy with any name you guys come up with.

**Comment by sheadawson** [comment]
I'm happy with $CurrentPage. If you could just drop the { down a line on that method to keep the code style I'll merge away.

**Comment by jonom** [comment]
Oops - I had it down originally, then got all confused when I was working on an older fork. Will do.

**Comment by jonom** [comment]
Done

**Comment by sheadawson** [comment]
Awesome thanks Jono!

**Pull request merged**